lib/frontend/components/GradientButton.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'dart:math' as math;
+
+class GradientButton extends StatelessWidget {
+  final String buttonText;
+  final Function buttonPressed;
+
+  static final LinearGradient gradientOutline = const LinearGradient(
```

Comment:
Use const instead of final for compile time assignments.  The difference between final and const is that the value of a final can be determined at runtime, while a const has to be set at compile time.  Since these styles are not determined during runtime, constants should be used for them.